docs/src/main/mdoc/typeclasses/bimonad.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: docs
+title:  "Bimonad"
+section: "typeclasses"
+source: "core/src/main/scala/cats/Bimonad.scala"
+scaladoc: "#cats.Bimonad"
+---
+# Bimonad
+
+The `Bimonad` trait directly extends `Monad` and `Comonad` without introducing new behaviour. `Bimonad` is 
+different of other `Bi` typeclasses like `Bifunctor`, `Bifoldable` or `Bitraverse` where the prefix describes a
+`F[_, _]`. The `Bimonad` is a `F[_]` and could be better seen as a dual monad i.e. something that is both a `Monad` and 
+a `Comonad`.
+
+If you use `Bimonad` as a convenience type such that `def f[T[_] : Monad, Comonad, S](fa: T[S]): S` is re-written to
+`def f[T[_] : Bimonad, S](fa: T[S]): S` keep in mind `Bimonad` has its own added laws so something that is both monadic 
+and comonadic may not necessarily be a lawful `Bimonad`.
+
+###Eval as a Bimonad
+Eval is a lawful `Bimonad` so you can chain computations and `extract` the result at the end.
+
+Note the equivalence:
+```scala mdoc
+import cats._
+import cats.data._
+import cats.implicits._
+
+Bimonad[Eval].pure(true).extract === Eval.now(true).value
```

Comment:
I think what you are trying to show here is that `pure` is implemented as `now` and `extract` as `value`? One alternative would be to define `Bimonad[Eval]` here just for demonstrative purposes, WDYT?

Reply:
> I think what you are trying to show here is that `pure` is implemented as `now` and `extract` as `value`?
Yes

> One alternative would be to define Bimonad[Eval] here just for demonstrative purposes, WDYT?

Something like that ?
```scala
val evalBimonad = Bimonad[Eval]

evalBimonad.pure(true).extract === Eval.now(true).value
```

Reply:
I was thinking like how they define `optionMonad` in the `Monad` docs:
http://typelevel.org/cats/typeclasses/monad.html